### PR TITLE
Fixes#199: Change the login URL on SignUp page

### PR DESCRIPTION
**Pull request by madhavrathi**
Fixes issue #199 

Changes: Now login url on signup works fine. 

Demo Link: http://urlfix.surge.sh/

@isuruAb @rishiraj824 @uday96 Please review and suggest any changes.

### CODE REVIEW

File: src/components/Auth/SignUp/SignUp.react.js [hunk 1]
```diff
@@ -131,81 +137,94 @@ export default class SignUp extends Component {
...
-        );
-    };
-}
-SignUp.propTypes = {
-    history: PropTypes.object
-}
+                        open={this.state.open}
+                        autoScrollBodyContent={true}
+                        onRequestClose={this.handleClose}>
+                        <div><Login /></div>
```
Comment (uday96) [inline src/components/Auth/SignUp/SignUp.react.js]: The login doesn't seem to work from the signup page. Kindly check @madhavrathi

**Comment by madhavrathi** [comment]
@isuruAb @uday96 I will fix this issue and update the PR.
Thanks

**Comment by madhavrathi** [comment]
@uday96 @rishiraj824 @isuruAb Please review the demo-link. 
http://urlfix.surge.sh/

**Comment by mariobehling** [comment]
Please resolve merge conflicts.

**Comment by madhavrathi** [comment]
@mariobehling The conflicts are fixed.
Thanks

**Pull request merged**